creator-node/src/apiSigning.js

```diff
@@ -85,11 +86,102 @@ const sortKeys = x => {
   return Object.keys(x).sort().reduce((o, k) => ({ ...o, [k]: sortKeys(x[k]) }), {})
 }
 
+/**
+ * Wrapper fn to perform basic validation that the requester is a valid SP and that the request came
+ * from the SP node itself. Uses the {spID, timestamp} as the input data to recover.
+ * @param {Object} data
```

Comment:
this jsdoc seems to have a ton of properties that aren't really relevant? like data, nodeEndpointFromSPFactory, delegateOwnerWalletFromSPFactory etc

Reply:
ah this comment is stale. updating